cli/src/main.rs

```diff
@@ -190,15 +190,27 @@ fn parse_resource_string(s: &str) -> Result<TrustResource> {
 enum Commands {
     /// Sub-command to configure trust store options, "trust --help for more details"
     Trust {
-        /// URL or path to file containing list of trust anchors in PEM format
+        /// URL or path to file containing list of user trust anchors in PEM format for CAWG validation
+        #[arg(long = "cawg_user_trust_anchors", env="C2PATOOL_CAWG_USERTRUST_ANCHORS", value_parser = parse_resource_string)]
+        cawg_user_trust_anchors: Option<TrustResource>,
+
+        /// URL or path to file containing list of CAWG trust anchors in PEM format for CAWG validation
+        #[arg(long = "cawg_trust_anchors", env="C2PATOOL_CAWG_TRUST_ANCHORS", value_parser = parse_resource_string)]
```

Comment:
@scouten-adobe  does CAWG have a distinction between trust anchors and user trust anchors?  I thought it was just a set of user supplied anchors for whatever domain was being used,